lambdas/build-stac/utils/stac.py

```diff
@@ -238,6 +237,10 @@ def get_assets_from_cmr(cmr_json, item) -> dict[pystac.Asset]:
             asset = gen_asset("documentation", link, item)
             if asset:
                 assets["documentation"] = asset
+
+    if None not in item.assets:
```

Comment:
Just to be sure I understand, if item.assets is truthy then use item.assets to override the data assets we just parsed?

Reply:
Reading the implementation above, does it make sense to make line 224 the same as line 220? Then you could just fill the "data" asset up on line 210 where assets is initialized. Thoughts? Just easier to understand what's going on by reducing state a very tiny bit.